modules/visual_script/visual_script_editor.cpp

```diff
@@ -42,6 +42,8 @@
 #include "visual_script_func_nodes.h"
 #include "visual_script_nodes.h"
 
+#include <iostream>
```

Comment:
Pretty sure this should not be here.

Reply:
Damn it, I always forget these.

Reply:
Weird to even have them there.
I use the Godot's print function or just a debugger instead(using a debugger is not always very easy though).